docs/releases.md

```diff
@@ -1,48 +1,68 @@
 # Release process
 
-The release process in broken up into 3 steps.
+## 1. Version and tag
 
-1. Increment the version and add a new tag
-2. Modify the changelog
-3. Merge the release branch and create a "github release"
+We use [Semantic Versioning](https://semver.org/#semantic-versioning-200).
 
-## 1. Increment the version and add a new tag
+1. Create a new working branch
+2. Run
 
-To update the version in package.json and create a tag run:
+   ```shell
+   $ yarn version
+   ```
 
-```shell
-$ yarn version
-```
+   This will update the `package.json` version, create a commit and add
+   an annotated version tag.
 
-The version number should be updated following [Semantic Versioning](https://semver.org/#semantic-versioning-200). The only change should be to package.json, and there should be a new tag on the commit prefixed with a "v".
+## 2. Update the changelog
```

Comment:
Since you know that what the new version of the package will be and the CHANGELOG entry, it would be possible to re-order the process to avoid the need for step 3. This would require some changes to the way `gulp changelog` works to output changes since the last release. The process would then be:

1. Run `gulp changelog` to get changes since last release and copy the output
2. Edit `CHANGELOG.md`, paste in the change list and edit as needed
3. Run `git add CHANGELOG.md` to stage the changelog for the next commit
4. Run `yarn version` to update the version number in `package.json`, commit the `package.json` and `CHANGELOG.md` updates and tag the result
5. Push the resulting commit and tag to GitHub

Reply:
> This would require some changes to the way gulp changelog works to output changes since the last release. 

That felt a little out of scope for the immediate term.